index.js

```diff
@@ -59,7 +59,7 @@ var parseOptions = module.exports._parseOptions = function (options) {
   var isProduction = process.env.NODE_ENV === "production";
   var isDevelopment = !isProduction;
   var servePath = (options.servePath || "assets");
-  var servePathPathname = parseUrl(servePath).pathname || "/";
+  var servePathPathname = url.parse(servePath, false, true).pathname || "/";
```

Comment:
-1 on deleting this method, a string of booleans aren’t very readable. Going to cherry-pick in your test, and then apply the above change to avoid deleting this method in this case, since what you’ve changed this to is equivalent to what’s written below.